engine/pom.xml

```diff
@@ -313,10 +313,11 @@
       ${camunda.artifact}.engine.*;${camunda.osgi.version};-noimport:=true
     </camunda.osgi.export.pkg>
     <camunda.osgi.import.defaults>
-      org.camunda.bpm.engine.variable.*;${camunda.osgi.import.camunda.commons.version},
+      org.camunda.bpm.engine.variable.*,
       org.camunda.commons.*;${camunda.osgi.import.camunda.commons.version},
     </camunda.osgi.import.defaults>
     <camunda.osgi.import.additional>
+      !com.google.gson.*,
```

Comment:
Could you quickly comment on why GSON should be excluded here? Did you mean to make it optional? If so, why?

Reply:
GSON has been shaded in this bundle

https://github.com/camunda/camunda-bpm-platform/blob/master/engine/pom.xml#L432

Reply:
Oh, yeah, you're right. Thanks for pointing me to it. 👍